Source/Widgets/Viewer/Viewer.js

```diff
@@ -703,7 +703,7 @@ Either specify options.terrainProvider instead or set options.baseLayerPicker to
         }
 
         cesiumWidget.screenSpaceEventHandler.setInputAction(pickAndSelectObject, ScreenSpaceEventType.LEFT_CLICK);
-        cesiumWidget.screenSpaceEventHandler.setInputAction(pickAndTrackObject, ScreenSpaceEventType.LEFT_DOUBLE_CLICK);
+        cesiumWidget.screenSpaceEventHandler.setInputAction(pickAndTrackObject, ScreenSpaceEventType.MIDDLE_CLICK);
```

Comment:
Keeping `LEFT_DOUBLE_CLICK` is fine for desktop.  For UX purposes, remember neither it nor single-middle-click is available on touch devices.  Double-middle-click and double-right-click should both be completely deprecated, as browser support is weak and even desktop users don't think to try them.